.github/workflows/publish-npm-packages.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish NPM Packages
+on:
+  workflow_dispatch:
+    inputs:
+      ampversion:
+        description: 'AMP version'
+        required: true
+      tag:
+        description: 'NPM package tag (latest | nightly)'
+        required: true
+jobs:
+  publish:
+    environment: NPM_TOKEN #TODO: change environment name in repo settings
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        extension:
+          [
+            'amp-selector',
+            'amp-accordion',
+            'amp-date-countdown',
+            'amp-date-display',
+            'amp-fit-text',
+            'amp-inline-gallery',
+            'amp-instagram',
+            'amp-lightbox',
+            'amp-timeago',
+            'amp-youtube',
+          ]
```

Comment:
For maintainability and visibility, I think this list should live in a file under the main extensions directory, say `extensions/published-extensions.json`, and be dynamically sourced in this yaml file using the [`fromJson()`](https://github.blog/changelog/2020-04-15-github-actions-new-workflow-features/) method supported by GH actions.

Reply:
Would argue that moving these to a new file makes this less visible, and using fromJSON() would require more steps ie. reading from a file, which could make it less maintainable as well.

Adding or removing components directly from the workflow is easy and clear.

Reply:
> Adding or removing components directly from the workflow is easy and clear.

The rationale behind my comment was to create a simpler workflow for a developer (possibly an external contributor) who wants to add a new bento component, at the cost of a tiny bit more complexity in this workflow file.

Additional benefits are:
- The author of a new bento extension should not have to know that this workflow file needs to be updated.
- The infra WG should not have to review a change to this workflow file each time a new component is added.

> moving these to a new file makes this less visible

Fair point. Once we start automatically generating `package.json` files, it may not be necessary to add a file at all. Something like the command below could be added as a step in this workflow, and its output could be used to dynamically create the workflow matrix.

```sh
find extensions/*/*/package.json | sed 's/extensions\///g' | sed 's/\/.*//g'
```
![image](https://user-images.githubusercontent.com/26553114/118733450-457dfb00-b80a-11eb-88e6-12f8de9d259f.png)

Anyhow, given that this isn't an immediately blocking issue, totally fine to punt and revisit when `package.json` files are auto-generated.

Reply:
I don't believe we should maintain another list at all, we already have one in `src/amphtml/build-system/compile/bundles.config.extensions.json`. Anything marked `npm: true` here is a Bento component opting into being published.

Reply:
Ok, good point to use the bundles config. I added a `setup` job to read from it and pass to `publish`. Here's how it looks https://github.com/estherkim/good-bot/actions/runs/857455380

Let me know if this works!

(re: package.jsons, I don't think we will be keep them in source, otherwise that might work too)

Reply:
> Here's how it looks [estherkim/good-bot/actions/runs/857455380](https://github.com/estherkim/good-bot/actions/runs/857455380)

Very nicely done!